applications/RomApplication/tests/structural_static_test_files/ProjectParametersROM.json

```diff
@@ -24,8 +24,8 @@
         },
         "rom_settings": {
             "nodal_unknowns": [
-                "DISPLACEMENT",
-                "2"
+                "DISPLACEMENT_X",
```

Comment:
i do believe that this information should be contained in the same file as the rom basis, so that one has all the information written within the file.

(see comment just above)

Reply:
done